components/ProgramsComponent.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+
+import React, { useState } from 'react';
+import { Box, Typography } from '@mui/joy';
+import OutlineButton from './OutlineButton';
+import LabelledIcon from './LabelledIcon';
+
+interface ProgramsComponentProps {
+  frontTitle: string;
+  frontImageSrc: string;
+  backContent: React.ReactElement[];
+  link: string;
+  description: string;
+}
+
+
+export default function ProgramsComponent({ frontTitle, frontImageSrc, backContent, link, description }: ProgramsComponentProps) {
+	const [isFlipped, setIsFlipped] = useState(false);
+
+  const toggleComponent = () => {
+		setIsFlipped(!isFlipped);
+	};
+
+  return (
+    <Box
+      onClick={toggleComponent}
+      className={'shadow border-round'}
+      style={{ 
+        width: '375px',
+        height: '425px',
+        overflow: 'hidden',
+      }}
+    >
+      {isFlipped ? (
+        <Box
+          style={{ 
+            padding: '40px',
+            display: 'flex',
+            flexDirection: 'column',
+            height: '100%',
+          }}
+        >
+          <Box
+            style={{ 
+              display: 'flex', 
+              justifyContent: 'center',
+            }}
+          >
+            <Typography className={'graphite'} level='body' style={{ textAlign: 'left', lineHeight: '1.2', }}>{description}</Typography>
```

Comment:
i think textAlign center might look a bit nicer and it looks more similar to the figma that way